src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -67,9 +86,28 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
         // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
         tabIndex={onRowClick ? 0 : undefined}
       >
+        {userReputationData && tokenInfoData && (
+          <div className={styles.reputationSection}>
+            <Icon
+              name="star"
+              appearance={{ size: 'extraTiny' }}
+              className={styles.icon}
+              title={MSG.starReputationTitle}
+              titleValues={{
+                reputation: userReputationData.userReputation,
+              }}
+            />
+            <Numeral
+              className={styles.reputation}
+              appearance={{ theme: 'primary' }}
+              value={userReputationData.userReputation}
+              unit={tokenInfoData.tokenInfo.decimals}
+            />
+          </div>
+        )}
         <div className={styles.section}>
           <UserAvatar
```

Comment:
While indeed the user avatar you've added has a user mention popover, please make it resemble the "new spec" of avatar popover that is currently on the Actions List:
- green hover over the avatar
- popover without an arrow
- popover offset

_(you can check `ActionsList` to get how it's implemented)_

Here's a demo:
![cs-ations-list-avatar-hover](https://user-images.githubusercontent.com/1193222/100226601-a2b68500-2f28-11eb-96c0-a0a3df8ef914.gif)